### ISSUE TITLE
@kubb/plugin-msw@3.0.0-beta.9 defaults generates incorrect import

### ISSUE THREAD

**Issue by allenthich** [kubb-labs/kubb#1359]
### What version of `kubb` is running?

3.0.0-beta.9

### What platform is your computer?

MacOS Ventura 13

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

```
"@kubb/cli": "^3.0.0-beta.9",
"@kubb/core": "^3.0.0-beta.9",
"@kubb/plugin-client": "^3.0.0-beta.9",
"@kubb/plugin-faker": "^3.0.0-beta.9",
"@kubb/plugin-msw": "^3.0.0-beta.9",
"@kubb/plugin-oas": "^3.0.0-beta.9",
"@kubb/plugin-react-query": "^3.0.0-beta.9",
"@kubb/plugin-ts": "^3.0.0-beta.9",
"@kubb/plugin-zod": "^3.0.0-beta.9"
```

### What steps can reproduce the bug?

1. Setup basic pets example with plugin-msw default config
```typescript
// kubb.config.ts
export default defineConfig({
  input: {
    path: './pets.yaml',
  },
  output: {
    path: './gen',
  },
  plugins: [
    pluginOas(),
    pluginTs(),
    pluginMsw(),
  ],
});
```
2. Run kubb generate
3. Notice generated `/mocks/*.ts` contain invalid relative imports (spaces)
```typescript
// mocks/createPetHandler.ts

import { CreatePetsMutationResponse } from "../ createPets .ts";
```

### How often does this bug happen?

Every time

### What is the expected behavior?

Default plugin-msw which uses 'data' parser should generate files that contain valid imports

### Swagger/OpenAPI file?

https://stackblitz.com/edit/stackblitz-starters-jlw6lq?file=pets.yaml

### Additional information

Hi, thanks for working on this awesome library.
I would be happy to make a pull request for this fix, but I'm unsure whether this is expected being a beta version and there are major breaking changes, ongoing refactors, etc.
I have not determined the cause of additional spacing within the import (exists in `faker.file.path`), but it appears the `faker` object is generated regardless of parser and renders the following import:

**plugin-msw/generators/mswGenerator.tsx**
```tsx
const faker = {
  file: getFile(operation, { pluginKey: [pluginFakerName] }),
  schemas: getSchemas(operation, { pluginKey: [pluginFakerName], type: 'function' }),
}

```
```tsx
{faker.file && faker.schemas.response && <File.Import name={[faker.schemas.response.name]} root={mock.file.path} path={faker.file.path} />}
```

**Playground:**
https://stackblitz.com/edit/stackblitz-starters-jlw6lq?file=gen%2Fhandlers%2FcreatePetsHandler.ts

**Comment by stijnvanhulle** [comment]
@allenthich Kubb v`3.0.0-beta.10` resolved your issue, can you have a look and see if that is the case?
Thanks for using Kubb :)

**Comment by allenthich** [comment]
@stijnvanhulle Thanks for the quick response! Confirmed fixed in `3.0.0-beta.10`.
